packages/web3-eth/src/web3_eth.ts

```diff
@@ -307,17 +1148,96 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
+	 *
+	 * ```ts
+	 * web3.eth.getPastLogs({
+	 *      address: "0x11f4d0A3c12e86B4b5F39B213F7E19D048276DAe",
+	 *      topics: ["0x033456732123ffff2342342dd12342434324234234fd234fd23fd4f23d4234"]
+	 *  }).then(console.log);
+	 * > [{
+	 *       data: '0x7f9fade1c0d57a7af66ab4ead79fade1c0d57a7af66ab4ead7c2c2eb7b11a91385',
+	 *       topics: ['0xfd43ade1c09fade1c0d57a7af66ab4ead7c2c2eb7b11a91ffdd57a7af66ab4ead7', '0x7f9fade1c0d57a7af66ab4ead79fade1c0d57a7af66ab4ead7c2c2eb7b11a91385']
+	 *       logIndex: 0n,
+	 *       transactionIndex: 0n,
+	 *       transactionHash: '0x7f9fade1c0d57a7af66ab4ead79fade1c0d57a7af66ab4ead7c2c2eb7b11a91385',
+	 *       blockHash: '0xfd43ade1c09fade1c0d57a7af66ab4ead7c2c2eb7b11a91ffdd57a7af66ab4ead7',
+	 *       blockNumber: 1234n,
+	 *       address: '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe'
+	 *   },
+	 *   {...}]
+	 *
+	 * web3.eth.getPastLogs(
+	 *     {
+	 *       address: "0x11f4d0A3c12e86B4b5F39B213F7E19D048276DAe",
+	 *       topics: ["0x033456732123ffff2342342dd12342434324234234fd234fd23fd4f23d4234"]
+	 *     },
+	 *     { number: FMT_NUMBER.NUMBER , bytes: FMT_BYTES.HEX }
+	 * ).then(console.log);
+	 * > [{
+	 *       data: '0x7f9fade1c0d57a7af66ab4ead79fade1c0d57a7af66ab4ead7c2c2eb7b11a91385',
+	 *       topics: ['0xfd43ade1c09fade1c0d57a7af66ab4ead7c2c2eb7b11a91ffdd57a7af66ab4ead7', '0x7f9fade1c0d57a7af66ab4ead79fade1c0d57a7af66ab4ead7c2c2eb7b11a91385']
+	 *       logIndex: 0,
+	 *       transactionIndex: 0,
+	 *       transactionHash: '0x7f9fade1c0d57a7af66ab4ead79fade1c0d57a7af66ab4ead7c2c2eb7b11a91385',
+	 *       blockHash: '0xfd43ade1c09fade1c0d57a7af66ab4ead7c2c2eb7b11a91ffdd57a7af66ab4ead7',
+	 *       blockNumber: 1234,
+	 *       address: '0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe'
+	 *   },
+	 *   {...}]
+	 * ```
+	 */
 	public async getPastLogs<ReturnFormat extends DataFormat = typeof DEFAULT_RETURN_FORMAT>(
 		filter: Filter,
 		returnFormat: ReturnFormat = DEFAULT_RETURN_FORMAT as ReturnFormat,
 	) {
 		return rpcMethodsWrappers.getLogs(this, filter, returnFormat);
 	}
 
+	/**
+	 * Gets work for miners to mine on. Returns the hash of the current block, the seedHash, and the boundary condition to be met (“target”).
+	 *
+	 * @returns The mining work as an array of strings with the following structure:
+	 *
+	 * String 32 Bytes - at index 0: current block header pow-hash
+	 * String 32 Bytes - at index 1: the seed hash used for the DAG.
+	 * String 32 Bytes - at index 2: the boundary condition (“target”), 2^256 / difficulty.
+	 *
+	 * ```ts
+	 * web3.eth.getWork().then(console.log);
+	 * > [
+	 *     "0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef",
+	 *     "0x5EED00000000000000000000000000005EED0000000000000000000000000000",
+	 *     "0xd1ff1c01710000000000000000000000d1ff1c01710000000000000000000000"
+	 * ]
+	 * ```
+	 */
 	public async getWork() {
 		return rpcMethods.getWork(this.requestManager);
 	}
 
+	/**
+	 * Used for submitting a proof-of-work solution.
```

Comment:
Should we mark that its not tested in 4.x

Reply:
Web3 doesn't do anything special with the data the user submits, so it's almost entirely up to the client the user is connected to whether it works or not